Directory.Build.props

```diff
@@ -0,0 +1,18 @@
+<Project>
+
+  <PropertyGroup>
+    <Product>YAXLib</Product>
+    <Authors>Sina Iravanian, Julian Verdurmen, axuno gGmbH and other contributors</Authors>
+    <Copyright>Copyright (C) Sina Iravanian, Julian Verdurmen, axuno gGmbH and other contributors</Copyright>
```

Comment:
```suggestion
    <Copyright>Copyright (C) YAXLib maintainers and other contributors</Copyright>
```

Reply:
Same as other comment. Will mark as resolved for now